github_connector/readme/USAGE.rst

```diff
@@ -85,6 +85,29 @@ Finally, you can download locally the source code of all your branches:
 
    .. image:: ../static/description/github_repository_branch_list.png
 
+Analysis source code
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+It's possible to create custom analaysis rule that can related to Organization and/or Repository to analyze repository code
+
+#. Go to 'Settings' / 'Analysis rule groups' and create records that allow to organize the rules
+
+#. Go to 'Settings' / 'Analysis rules' and create rules, for example:
+
+1. Name: All code, Group: General, Paths: ``*``
+2. Name: Python code, Group: General, Paths: ``*.py``
+3. Name: Xml code, Group: General, Paths: ``*.xml``
+4. Name: Repository 1, Group: Custom, Paths: /path/
+
+Note: Paths field in 'Analysis rule' allow to put multiple paths for line, path format is according to https://git-scm.com/docs/gitignore#_pattern_format
+
+#. Go to 'Organizations' and define Analysis rules (optional)
+#. Go to 'Repositories' and define Analysis rules (optional)
```

Comment:
Full menu path and include branches:

```suggestion
#. Go to *GitHub > GitHub Community > Organizations* and define Analysis rules (optional)
#. Go to *GitHub > GitHub Repository > Repositories* and define Analysis rules (optional)
#. Go to *GitHub > GitHub Repository > Repository Branches* and define Analysis rules (optional)
```

Reply:
Now I'm seeing that functionally there's no option of putting rules at branch level, but it should be, as for example folder organization has changed over the branches (versions - OCB 7.0 has /openerp/ folder instead of /odoo)

Reply:
Maybe this problem can solved with **github_connector_odoo** and `module_paths` field in Repository Branch. https://github.com/OCA/interface-github/pull/64